config/default.yml

```diff
@@ -674,6 +674,19 @@ PreCommit:
       - '**/*.gemspec'
       - '**/*.rb'
 
+  RubySyntax:
+    enabled: false
+    description: 'Check ruby syntax'
+    required_executable: 'ruby'
+    command: [
+      'ruby',
+      '-e',
+      'ARGV.each { |applicable_file| ruby_c_output = `ruby -c #{applicable_file}`; puts ruby_c_output if ruby_c_output != "Syntax OK" }'
```

Comment:
Judging success based on outputted text is potentially brittle if Ruby ever changes the output for a failed parse.

More robust would be to depend on the exit status of the binary, as a status of `0` means "success" by convention. 

For example:

```ruby
ARGV.each { |applicable_file| output = `ruby -c #{applicable_file}`; puts output unless $?.success? }
```